coalib/bears/Bear.py

```diff
@@ -251,13 +251,15 @@ def execute(self, *args, **kwargs):
             return [] if result is None else list(result)
         except (Exception, SystemExit):
             if self.kind() == BEAR_KIND.LOCAL:
-                self.warn('Bear {} failed to run on file {}. Take a look '
-                          'at debug messages (`-V`) for further '
-                          'information.'.format(name, args[0]))
+                if '-V' not in args:
+                    self.warn('Bear {} failed to run on file {}. Take a look '
+                              'at debug messages (`-V`) for further '
+                              'information.'.format(name, args[0]))
             else:
-                self.warn('Bear {} failed to run. Take a look '
-                          'at debug messages (`-V`) for further '
-                          'information.'.format(name))
+                if '-V' not in args:
```

Comment:
using ```self.log_level()``` shows error-
There is not attribute named log_level in LocalBear or GlobalBear